src/mongoc/mongoc-stream.h

```diff
@@ -61,6 +61,7 @@ struct _mongoc_stream_t
    ssize_t          (*poll)            (mongoc_stream_poll_t *streams,
                                         size_t                nstreams,
                                         int32_t               timeout);
+   void             (*failed)          (mongoc_stream_t *stream);
```

Comment:
The padding seems off by one already, it was not decremented with the addition of poll.
Should I decrement it for poll in a separate commit?